pinning.go

```diff
@@ -693,15 +696,18 @@ func (s *Server) handleAddPin(e echo.Context, u *util.User) error {
 	if err != nil {
 		return err
 	}
-	return e.JSON(http.StatusAccepted, status)
+	return e.JSON(http.StatusOK, status)
```

Comment:
@snissn the IPFS spec expects a 202 and not a 200 - https://ipfs.github.io/pinning-services-api-spec/#operation/addPin

Reply:
Thank you! I was confused by the 202, i'll update this. Thanks for catching!